### DISCUSSION TITLE
create step implementation fail

### DISCUSSION THREAD

**Discussion by gaomeizi** [getgauge/gauge#1840] [General]
Unable to generate implementation. Error: Connection got disposed.
![image](https://user-images.githubusercontent.com/29977475/103719052-27ee8880-5003-11eb-95b9-c74801dfcfa0.png)

python,vscode,gauge

Gauge version: 1.1.3
Commit Hash: 9d3a3d5

Plugins
-------
html-report (4.0.12)
python (0.3.11)
screenshot (0.0.1)

**Comment by NivedhaSenthil** [comment]
please try upgrading your Gauge version and other plugin versions and let us know if you are still facing the issue ?
